### ISSUE TITLE
Possible backward driving while follow lane and waypoints

### ISSUE THREAD

**Issue by wanbill** [carla-simulator/carla#6701]
Hi Carla teams, I am currently playing around with Carla. I am wondering if the Python API file or possibly the agent file is able to support navigation through backward driving. I try to set control. gear = -1 and control. reverse to true but the car did not move as I am expecting.
CARLA version: 0.9.12
Platform/OS: ubuntu 20.04
Best

**Comment by Ezharjan** [comment]
> Hi Carla teams, I am currently playing around with Carla. I am wondering if the Python API file or possibly the agent file is able to support navigation through backward driving. I try to set control. gear = -1 and control. reverse to true but the car did not move as I am expecting. CARLA version: 0.9.12 Platform/OS: ubuntu 20.04 Best

Yeah, I also found that backward driving only works in `manual_control`. And the `vehicle_control. gear = -1` actually means `vehicle is running on back gear`. 
**Did you found any solution to this?**
